src/GridLayoutManager.ts

```diff
@@ -9,26 +9,29 @@ export class GridLayoutManager extends WrapGridLayoutManager {
   private _getSpan: (index: number) => number;
   private _isGridHorizontal: boolean | undefined;
   private _renderWindowSize: Dimension;
+  private _acceptableRelayoutDelta: number;
   constructor(
     layoutProvider: LayoutProvider,
     renderWindowSize: Dimension,
     getSpan: (index: number) => number,
     maxSpan: number,
+    acceptableRelayoutDelta: number,
     isHorizontal?: boolean,
     cachedLayouts?: Layout[],
   ) {
     super(layoutProvider, renderWindowSize, isHorizontal, cachedLayouts);
     this._getSpan = getSpan;
     this._isGridHorizontal = isHorizontal;
     this._renderWindowSize = renderWindowSize;
+    this._acceptableRelayoutDelta = acceptableRelayoutDelta;
```

Comment:
This should always be greater than 0. Consider throwing an exception if it isn't